frontend/public/components/namespace-menu-toggle.tsx

```diff
@@ -0,0 +1,99 @@
+import * as React from 'react';
+import { MenuToggle, Popper } from '@patternfly/react-core';
```

Comment:
I'm not sure we should be using PatternFly's version of Popper. I don't think we're using this component elsewhere. We use our own Popper implementation in the console-shared package. Here's an example of where we use it elsewhere: https://github.com/openshift/console/blob/master/frontend/packages/console-shared/src/components/kebab/KebabMenu.tsx

Reply:
I was following the pattern PatternFly has recommended when combining the MenuToggle and Menu component : https://www.patternfly.org/v4/demos/composable-menu#basic-menu-with-toggle

I did try to implement using the custom Popper implementation there were a couple of issues.  First it didn't control the width of the menu.  The PF version will set the width of the menu to the width of the MenuToggle if the menu is attached to the MenuToggle.  Without this we would have to force or calculate the width of the menu or it runs off screen on smaller screen widths.

Secondly on certain screen sizes (both on smaller widths and/or smaller heights), the menu would display in a strange position - often going partially off screen to the left and top. I tried all the placement options but the behavior still existed.  To ensure a logical tab order, I had popper put the menu in the HTML immediately MenuToggle, so *maybe* the placement in the HTML was confusing the position logic.

Keeping the PF Popper because it is recommended by PF and the display issues with the custom popper component.